projects/plugins/boost/app/assets/src/js/features/speed-score/pop-out/pop-out.tsx

```diff
@@ -69,14 +74,21 @@ function PopOut( { scoreChange }: Props ) {
 	 * Dismissed means that the user asked to never show us this alert again.
 	 */
 	const [ isDismissed, dismissAlert ] = useDismissibleAlertState( message.id );
-
 	/*
 	 * Hide the alert for now. The alert will show up again if the user refreshes the page.
 	 */
 	const [ isClosed, setClose ] = useState( false );
 
 	const hideAlert = () => setClose( true );
 
+	if ( hasScoreChanged ) {
+		recordBoostEvent( 'speed_score_alert_shown', {
+			score_direction: scoreChange > 0 ? 'up' : 'down',
+			mobile_score: scores.mobile,
+			desktop_score: scores.desktop,
```

Comment:
I can see how `score_direction` is useful. But I am not sure the mobile and desktop scores are of any use. I am thinking about the way the tracks interface works. Usually, the properties are used as buckets. With scores ranging widely, I don't see them usable as buckets.

Reply:
Thanks. I was thinking about that, but I left this in to see how it would look. You're right, it wouldn't be useful.